apps/console/src/features/groups/components/edit-group/edit-group-roles.scss

```diff
@@ -0,0 +1,5 @@
+.list-group-roles-section {
```

Comment:
shall we add a header to this file as well

Reply:
Fixed with 5f3e39c54d9a02658bf4c6d7abd3f538899828ea